.github/workflows/ci-workflow.yml

```diff
@@ -234,6 +248,16 @@ jobs:
           java-version: '8'
       - name: Build with Maven
         env:
-          MAVEN_OPTS: -Xmx2g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN -Dmaven.surefire.arguments="-Duser.language=fr -Duser.region=FR -Duser.timezone=Europe/Paris" -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
+          MAVEN_OPTS: -Xmx2g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN -Duser.language=fr -Duser.country=FR -Duser.region=FR -Duser.timezone=Europe/Paris -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
         run: |
           mvn -V -T 0.7C package -B -Ddir-only -ntp -ff -pl -nifi-assembly -pl -nifi-system-tests -nsu
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ false }}
```

Comment:
There is a problem with artifact upload on Windows.  The upload succeeded when run against a small subset of the project, but failed 100% of time against full project.

Example:
https://github.com/greyp9/nifi/runs/2459360376?check_suite_focus=true

`
ENOENT: no such file or directory, realpath 'D:\a\nifi\nifi\nifi-nar-bundles\nifi-standard-bundle\nifi-jolt-transform-json-ui\target\frontend-working-directory\node\node_modules\npm\node_modules\libnpx\node_modules\yargs\node_modules\cliui\node_modules\string-width\node_modules\is-fullwidth-code-point\node_modules'
`
It seems that `nifi-nar-bundles\nifi-standard-bundle\nifi-jolt-transform-json-ui` is creating paths that exceed the 256 character limit for Windows paths, and that this new job cannot handle them.  It is possible that this issue contributes to the instability of the Github CI Windows job.

Reply:
Yep lets just punt on that archival on windows for now then - makes sense.

Reply:
`nifi % find . -type d -name node_modules`